modules/oc-mirror-workflows-table.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-workflows-table_{context}"]
+= Comparison of oc-mirror workflows
+
+Use the following table to compare the supported use cases for the mirror-to-disk (m2d), disk-to-mirror (d2m), and mirror-to-mirror (m2m) workflows.
+
+[%header,cols="3",cols="2,1,1",options="header"]
```

Comment:
Consider adding the subs attribute to this block because of the bold formatting.

> [%header,cols="3",cols="2,1,1",options="header",subs="quotes"]